paper/paper.md

```diff
@@ -111,6 +115,20 @@ When the MCTS solver is accurately selecting the optimal solutions, it will cont
 
 ![Convergence of visits.](gw_visits.png?raw=true "Title")
 
+# Real World Applications
+
+*mctreesearch4j* is designed to run on the JVM, and therefore is useful in a research setting where JVM is intended to be used alongside either the implementation of the game of the core AI engine. Typically, for state-of-the-art AI, MCTS is used in combination with other Deep Learning algorithms, where MCTS drives the exploration of hypothetical games to be played out in order for the agent to make better decisions.
```

Comment:
I dislike the term state-of-the-art since it's so subjective but I'll leave it up to you.

Reply:
Its a pretty common term in research i find.